compiler/record-minmax/include/RecordMinMax.h

```diff
@@ -58,6 +61,8 @@ class RecordMinMax
   luci_interpreter::Interpreter *getInterpreter() const { return _interpreters[0].get(); }
   MinMaxObserver *getObserver() const { return _observers[0].get(); }
 
+  std::vector<std::vector<std::vector<char>>> importH5Data(const std::string &input_data_path);
```

Comment:
plz do not use 3D vector but use logical meaning of it, with `using` alias

Reply:
I think the below aliasing is ok.

std::vector<char> -> Buffer
std::vector<Buffer> -> InputData
std::vector<InputData> -> InputDataset

Reply:
Done